### ISSUE TITLE
[Enhancement]: Bulk Local User Creation and Account Updates

### ISSUE THREAD

**Issue by TankParts** [advplyr/audiobookshelf#2271]
### Describe the feature/enhancement

The feature is to be able to create and update users and passwords in bulk via uploading a csv or a similar method. This would be useful for administrators who need to manage a large number of users and want to avoid creating or updating them one by one. 

The feature could work as follows:

- The addmin uploads a csv file with one column per user setting. eg: username, password, email, account type, can download, can update. etc. 
- The server validates the csv file and checks for any errors or duplicates.
- The server creates or updates the users according to the csv file, and sends a confirmation message admin.
- The administrator can view the status of the user creation or update process in the dashboard.

I think this feature would enhance the usability and functionality of Audiobookshelf, and make it easier for admins to manage their users. Thank you for your consideration.

**Comment by nichwall** [comment]
I agree user management would need to be better for large servers/admin tasks.

There is a related FR here (not necessary to complete it first, but should have a better UI before allowing mass creation of users) https://github.com/advplyr/audiobookshelf/issues/2066

I wonder if there should be another "group" column on users to more easily manage users, but will need to think about it more. That easily turns into a rabbit hole.

In the meantime, you could use the ABS API (documented here https://api.audiobookshelf.com) to create/update/read users from/to a CSV or spreadsheet
